src/aftereffects/OpenColorIO_AE.cpp

```diff
@@ -634,14 +634,14 @@ static PF_Err DoRender(
                 
                 if(arb_data->source == OCIO_SOURCE_ENVIRONMENT)
                 {
-				#ifdef MAC_ENV
-					char *envFile = std::getenv("OCIO");
-					const bool gotOCIO = (envFile != NULL);
-				#else
-					char envFile[32767] = { '\0' };
-					const DWORD envResult = GetEnvironmentVariable("OCIO", envFile, 32767);
-					const bool gotOCIO = (envResult > 0);
-				#endif
+                #ifdef MAC_ENV
```

Comment:
Could you use/enhance the getenv() from platform.h/cpp ?

Reply:
Hmmm, I never saw that. One problem is OCIO::Platform::getenv() is not part of the OCIO API, but used by the library internally. Maybe that's why ociobakelut and other programs we include don't use it.

Seems like there should be an API call to get the environment config. Or at least a constant with "OCIO".

Reply:
So I figured if I couldn't use OCIO::Platform::getenv() I should at least make my own rather than have all those #ifdefs floating around. Couple more commits made.

Reply:
Good idea.